content/600-about/200-prisma-docs/30-docs-components/01-mdx-examples.mdx

```diff
@@ -286,6 +302,26 @@ async function main() {
 ```
 ````
 
+## FootNote
```

Comment:
How do I put the link to the footnote in the main text itself? Manually? Does the footnote block have an anchor I can link to?

Reply:
You can find an example usage here: /200-concepts/100-components/03-prisma-migrate/100-supported-types-and-db-features.mdx

Reply:
Ok, so put it there manually and no link in the example usage.
Might be worth to mention that here as well so it is clear.